tests/chainer_tests/functions_tests/activation_tests/test_softplus.py

```diff
@@ -1,80 +1,56 @@
-import unittest
-
 import numpy
 
-import chainer
-from chainer.backends import cuda
 from chainer import functions
-from chainer import gradient_check
 from chainer import testing
-from chainer.testing import attr
 
 
 @testing.parameterize(*testing.product({
     'shape': [(3, 2), ()],
     'dtype': [numpy.float16, numpy.float32, numpy.float64],
 }))
 @testing.fix_random()
-class TestSoftplus(unittest.TestCase):
+@testing.inject_backend_tests(
+    None,
+    # CPU tests
+    [
+        {},
+        {'use_ideep': True},
```

Comment:
Does Softplus use ideep if it's available?

Reply:
Yeah you are right, It does not use ideep.